app/Renderers/Layouts/Mention/Mention.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Renderers\Layouts\Mention;
+
+use App\Renderers\Layouts\Columns\Standard;
+use App\Renderers\Layouts\Layout;
+
+class Mention extends Layout
+{
+    /**
+     * Available columns
+     * @return array[]
+     */
+    public function columns(): array
+    {
+        $columns = [
+            'name' => [
+                'key' => 'name',
+                'label' => 'entities/mentions.fields.element',
+                'render' => Standard::ENTITYLINK,
+            ],
+            'type' => [
+                'key' => 'type',
+                'label' => 'crud.fields.type',
+                'render' => function ($model) {
+                    return __('entities.' . $model->entity->type());
```

Comment:
Crashed when a mention is to a campaign, and wrong when it's linked to a quest elements, timeline element, or post